packages/sdk/src/client/ecdsa/actions/account.ts

```diff
@@ -0,0 +1,141 @@
+import { type Account, type Address, type Chain, type Client, getAddress, type Hash, type Hex, parseAbi, parseEventLogs, type Prettify, toHex, type TransactionReceipt, type Transport } from "viem";
+import { readContract, waitForTransactionReceipt, writeContract } from "viem/actions";
+import { getGeneralPaymasterInput } from "viem/zksync";
+
+import { FactoryAbi } from "../../../abi/Factory.js";
+import { type CustomPaymasterHandler } from "../../../paymaster/index.js";
+import { encodeModuleData, encodeSession } from "../../../utils/encoding.js";
+import { noThrow } from "../../../utils/helpers.js";
+import type { SessionConfig } from "../../../utils/session.js";
+
+export type DeployAccountArgs = {
+  owner: Address; // Wallet owner
+  contracts: {
+    accountFactory: Address;
+    session: Address;
+  };
+  initialSession?: SessionConfig;
+  salt?: Uint8Array; // Random 32 bytes
+  prefix?: string; // vendor prefix
+  onTransactionSent?: (hash: Hash) => void;
+  paymasterHandler?: CustomPaymasterHandler;
+  paymaster?: {
+    address: Address;
+    paymasterInput?: Hex;
+  };
+};
+
+export type DeployAccountReturnType = {
+  address: Address;
+  transactionReceipt: TransactionReceipt;
+};
+
+export type FetchAccountArgs = {
+  prefix?: string; // vendor prefix
+  owner: Address; // Wallet owner
+  contracts: {
+    accountFactory: Address;
+    session: Address;
+  };
+};
+
+export type FetchAccountReturnType = {
+  address: Address;
+  owner: Address;
+};
+
+const NULL_ADDRESS = "0x0000000000000000000000000000000000000000";
+
+export const deployAccount = async <
```

Comment:
Do we really need duplicate deployAccount function?

Reply:
yeah, because deployAccount from passkeyClient initializes the account with the passkey validator and passkey public key